include/aspect/simulator.h

```diff
@@ -1878,6 +1878,10 @@ namespace aspect
 
       mutable TimerOutput                 computing_timer;
 
+      // We define a timer so that we can sum up the wall times even on a restart
+      Timer walltime;
+      // A variable that is the total wall time that has elapsed up to the last snapshot
+      double total_walltime_since_last_snapshot;
```

Comment:
Please make the following changes. The different comments will make sure that our documentation system doxygen can interpret this correctly as the description of a member variable, and I think the current name is still not making clear what this variable actually is (take care to rename the other places where this variable is used):

```suggestion
      /**
       * A timer that we use to sum up the wall time between restarts.
       */
      Timer walltime;
      
      /**
       * The total wall time that has elapsed up to the last snapshot
       * that was created.
       */
      double total_walltime_until_last_snapshot;
```

Reply:
Ok thanks